src/components/TextWithHighlight/TextWithHighlight.tsx

```diff
@@ -1,15 +1,23 @@
-import React, { useRef, forwardRef, useMemo, useLayoutEffect } from "react";
-import PropTypes from "prop-types";
+import React, { forwardRef, useLayoutEffect, useMemo, useRef } from "react";
 import cx from "classnames";
 import Tooltip from "../../components/Tooltip/Tooltip";
 import useIsOverflowing from "../../hooks/useIsOverflowing";
 import useMergeRefs from "../../hooks/useMergeRefs";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import { DialogPositions } from "../../constants";
 import "./TextWithHighlight.scss";
 
-const getTextPart = (text, key, shouldHighlight, wrappingTextTag = "em", wrappingElementClassName) => {
+const getTextPart = (
+  text: string,
+  key: number,
+  shouldHighlight: boolean,
+  wrappingTextTag = "em",
+  wrappingElementClassName: string
+) => {
   const WrappingElement = wrappingTextTag;
   if (shouldHighlight) {
     return (
+      // @ts-ignore variable-component
       <WrappingElement className={cx("highlight-text", wrappingElementClassName)} key={key}>
```

Comment:
I am thinking there is a typescript way to implement this without ts ignore. let's check?

Reply:
I'll check

Reply:
Fixed 👍